discord/bot.py

```diff
@@ -43,4 +53,31 @@ async def ask(
     await ctx.send(args, view=Buttons())
 
 
-bot.run(os.environ["TOKEN"])
+# wywołanie bota i przejście do rozmowy prywatnej
```

Comment:
Jak piszemy kod po angielsku, to komentarze też po angielsku 😝  btw. komentarz raczej tutaj jest nie potrzebny. Bo nazwa funkcji mówi sama za siebie. 

Zmien komendę na małe litery imo. I tak nazwy komend będziemy wymyślać jeszcze. 
[StackOverflow Best practieces for writing good comments] (https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/)